src/deciderjob/agent_tests/Functional/schedulerTest.php

```diff
@@ -225,7 +226,8 @@ private function runnerDeciderScanWithNoEventsAndOnlyNomosShouldNotMakeADecision
     $this->dbManager->queryOnce("INSERT INTO highlight (fl_fk,start,len) VALUES(12222,12,3)");
     $this->dbManager->queryOnce("INSERT INTO highlight (fl_fk,start,len) VALUES(12222,18,3)");
 
-    list($success,$output,$retCode) = $runner->run($uploadId=2, $userId=6, $groupId=4, $jobId=31, $args="");
+    /* uploadId 1 includes pfile 4 */
+    list($success,$output,$retCode) = $runner->run($uploadId=1, $userId=2, $groupId=2, $jobId=31, $args="");
```

Comment:
Why do you change every `$userId` to `2` and every `$groupId` to `2`? I do not see the change in the setup which would explain these changes.